test/javascripts/acceptance/chat-quoting-test.js

```diff
@@ -197,10 +197,6 @@ acceptance(
         exists("#reply-control.composer-action-createTopic"),
         "the composer does not open"
       );
-      assert.ok(
```

Comment:
Not really possible to test this now with the way the clipboard promise is structured, we skip copying to clipboard in tests anyway